### PR TITLE
Alphabetize API methods and edit for consistency

### LINKED ISSUE #363: [Feature Request] Sort APIs alphabetically
It'd be nice to have the APIs sorted alphabetically. If there is a reason not to do that, please state it / make sections.

See https://besu.hyperledger.org/en/stable/Reference/API-Methods/#eth_accounts

### PR DESCRIPTION
**Pull request by alexandratran**
## Pull request checklist

Use the following list to make sure your PR fits the Besu doc quality standard.

### Before creating the pull request

Make sure that:

- [x] [all commits in this PR are signed off for the DCO](https://wiki.hyperledger.org/display/BESU/DCO).
- [x] you read the [contribution guidelines](https://wiki.hyperledger.org/display/BESU/Contributing+to+documentation).
- [x] you have [tested your changes locally](https://wiki.hyperledger.org/display/BESU/MkDocs+And+Markdown+Guide#MkDocsAndMarkdownGuide-PreviewTheDocumentation) before submitting them to the community for review.

### After creating your pull request and tests finished

Make sure that:

- [ ] you fixed all the issues raised by the tests, if any.
- [ ] you verified the rendering of your changes on [ReadTheDocs.org PR preview](https://wiki.hyperledger.org/display/BESU/MkDocs+And+Markdown+Guide#MkDocsAndMarkdownGuide-PreviewwithReadTheDocs)
  and updated the testing link (see [Testing](#testing)).

## Describe the change

- alphabetize API methods
- edit parameters/results styles across APIs for consistency
- no changes to the examples

## Issue fixed

fixes #363 

### For content changes

- [x] Doc content
- [ ] Doc pages organisation

### For tools changes

- [ ] CircleCI workflow
- [ ] Build and QA tools (lint, vale,…)
- [ ] MkDocs templates
- [ ] MkDocs configuration
- [ ] Python dependencies
- [ ] Node dependencies and JavaScript
- [ ] ReadTheDocs configuration
- [ ] GitHub integration

### CODE REVIEW

**Comment by bgravenorst** [comment]
General comment: There's still a bit of inconsistency with our usage of full stops.

File: CI/requirements.txt [hunk 1]
```diff
@@ -3,7 +3,6 @@ mkdocs-material==6.1.7
 Markdown==3.2.1
 markdown-fenced-code-tabs==1.0.3
 markdown-include==0.5.1
-MarkupSafe==1.1.1
```
Comment (bgravenorst) [inline CI/requirements.txt:6]: I made this change in a previous PR, so this may cause a conflict.

**Review by bgravenorst — APPROVED** [review]
LGTM

**Review by NicolasMassart — APPROVED** [review]
Looks good to me.

**Pull request merged**